src/lib.rs

```diff
@@ -272,6 +272,34 @@ where
     }
 }
 
+#[cfg(any(feature = "log", feature = "tracing"))]
+macro_rules! default_impl_skip_error_iterator {
+    ($method_name:ident,$log_level:expr) => {
+        #[doc = concat!(
+                            "Shortcut for [`SkipError::skip_error_and_log`] with a log level of [`",
+                            stringify!($log_level),
+                            "`].\n\n",
+                            "For example\n",
+                            "```edition2018\n",
+                            "use skip_error::SkipError;\n",
+                            "# fn main() {\n",
+                            "# testing_logger::setup();\n",
+                            "vec![Ok(1), Ok(2), Err(\"'three' is not a valid number\"), Ok(4)]\n",
+                            "  .into_iter()\n",
+                            "  .skip_error_and_debug()\n",
+                            "  .collect::<Vec<_>>();\n",
+                            "testing_logger::validate(|captured_logs| {\n",
+                            "  assert!(captured_logs[0].body.contains(\"'three' is not a valid number\"));\n",
```

Comment:
Not sure about this add (would be nice to test):
```suggestion
                            "  assert!(captured_logs[0].body.contains(\"'three' is not a valid number\"));\n",
                            "  assert_eq!(captured_logs[0].level, ", stringify!($log_level), ");\n",
```

Reply:
This is not gonna work because `test_logger` expect a `log_level` of type `log::Level`. So when `tracing`'s feature is activated, `log_level` is going to be of type `tracing::Level`. But I can do something like I did with the previous macro, taking 3 arguments instead of 2 (an additional one for `expected_log_level`).

Reply:
Indeed, I suspected that. Well, as you wish (this test is not that important)!

Reply:
Done in 318cd10.